client/src/components/pages/application/Signup/index.js

```diff
@@ -1,5 +1,195 @@
 import React from 'react';
+import axios from 'axios';
+import Alert from '../../../common/Alert';
+import Card from '../../../common/card';
+import Typography from '../../../common/Typography';
+import TextField from '../../../common/TextField';
+import Button from '../../../common/Button';
+import signupValidate from '../../../../utils/application/SignupValidation';
+import SelectBox from '../../../application/SelectBox';
+import signupImage from '../../../../assets/images/signup.svg';
+import './index.css';
 
-const SignUp = () => <h1>SignUp</h1>;
+const initialState = {
+  fullName: '',
+  email: '',
+  password: '',
+  passwordConfirmation: '',
+  location: '',
+  message: [],
```

Comment:
I think message could be string that it will display a specific line so in my opioin there is no need to use arrays .

Reply:
This message is coming from validation yup, and it's coming as an array, also I used includes below for the message and can't use it for string, I still think in a better way to solve this problem.

Reply:
ok, np, as I said you can use `notistack` from material ui for notifications
https://github.com/iamhosseindhv/notistack